### PR TITLE
[doc](update-key)add update key doc

### PR DESCRIPTION
**Pull request by JNSimba**
## Problem summary
At present, only the value column can be modified through the update command, 
and the modification of the key column can be done by flinkcdc synchronization
## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by dataroaring — APPROVED** [review]
LGTM

File: docs/en/docs/data-operate/update-delete/update.md [hunk 1]
```diff
@@ -123,6 +123,9 @@ Since the table order is a UNIQUE model, the rows with the same Key, after which
   |---|---|---| 
   | 1 | 100 | Pending shipments |
 
+## Update primary key column data
+Currently, the Update operation only supports updating the Value column, and the update of the Key column can refer to [Using FlinkCDC to update key column data synchronously](../../ecosystem/flink-doris-connector.md#Use-FlinkCDC-to-update-Key-column-data-synchronously)
+
```
Comment (dataroaring) [inline docs/en/docs/data-operate/update-delete/update.md:128]: Update primary key column is enough.

File: docs/en/docs/ecosystem/flink-doris-connector.md [hunk 2]
```diff
@@ -441,6 +441,16 @@ WITH (
 insert into doris_sink select id,name from cdc_mysql_source;
 ```
 
+## Use FlinkCDC to update Key column data synchronously
+Generally, in a business database, the number is used as the primary key of the table, such as the Student table, the number (id) is used as the primary key, but with the development of the business, the number corresponding to the data may change.
```
Comment (dataroaring) [inline docs/en/docs/ecosystem/flink-doris-connector.md:445]: same as above.

File: docs/zh-CN/docs/data-operate/update-delete/update.md [hunk 3]
```diff
@@ -120,6 +120,9 @@ Query OK, 1 row affected (0.11 sec)
   |---|---|---| 
   | 1 | 100 | 待发货 |
 
+## 更新主键列数据
+目前Update操作只支持更新Value列，Key列的更新可参考[使用FlinkCDC同步更新Key列数据](../../ecosystem/flink-doris-connector.md#使用FlinkCDC同步更新Key列数据)
+
```
Comment (dataroaring) [inline docs/zh-CN/docs/data-operate/update-delete/update.md:125]: same as above

File: docs/zh-CN/docs/ecosystem/flink-doris-connector.md [hunk 4]
```diff
@@ -433,6 +436,16 @@ WITH (
 insert into doris_sink select id,name from cdc_mysql_source;
 ```
 
+## 使用FlinkCDC同步更新Key列数据
```
Comment (dataroaring) [inline docs/zh-CN/docs/ecosystem/flink-doris-connector.md:439]: same as above

**Review by dataroaring — CHANGES_REQUESTED** [review]

**Comment by dataroaring** [comment]
run buildall

**Comment by luzhijing** [comment]
LGTM

**Review by luzhijing — DISMISSED** [review]

**Comment by dataroaring** [comment]
run build all

**Comment by dataroaring** [comment]
run buildall

**Review by zhannngchen — APPROVED** [review]
LGTM

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**